doc/api/documentation.markdown

```diff
@@ -66,3 +66,19 @@ Every HTML file in the markdown has a corresponding JSON file with the
 same data.
```

Comment:
Ah my bad. I just saw few linkable text on this page and reached out for your help. Perhaps we can leave this page as nothing much is needed here.